eo-maven-plugin/src/main/java/org/eolang/maven/OptimizationTask.java

```diff
@@ -144,15 +144,18 @@ private Path make(final XML xml, final Path file) throws IOException {
      * @param tojo Tojo
      * @param common Optimization
      * @return Optimization for specific Tojo
+     * @throws Exception If fails
      */
-    private Optimization optimization(final ForeignTojo tojo, final Optimization common) {
+    private Optimization optimization(final ForeignTojo tojo, final Optimization common)
```

Comment:
@Yanich96 Let's now take a look on the pseudo logic of `OptimizationTask` and `OptCached` classes and how they work:
```java
//OptimizationTask start
if(tojo.hasHash()){
   //OptCached internals start
   if(contains(in_cache)){
      return optimization from cache
   } else {
      run "common" optimization
   }
  //OptCacjed internals end
} else {
   run "common" optimization 
}
//OptimizationTask end
```
What if we simplify it to a something like the following:

```java
if(tojo.hasHash() &&  contains(in_cache)) {
      return optimization from cache
} else {
   run "common" optimization 
}
```

What do you think?

Reply:
@volodya-lombrozo 
I will create issue